frontend/packages/noobaa-storage-plugin/src/components/buckets-card/buckets-card-item.tsx

```diff
@@ -1,75 +1,120 @@
 import * as React from 'react';
-import * as _ from 'lodash';
+import { Link } from 'react-router-dom';
+import { InProgressIcon } from '@patternfly/react-icons';
 import { RedExclamationCircleIcon } from '@console/shared';
-import { LoadingInline, pluralize, humanizeNumber } from '@console/internal/components/utils';
+import { humanizeNumber, pluralize } from '@console/internal/components/utils';
 
 const formatCount = (count: number) => {
   const hCount = humanizeNumber(count);
   return `${hCount.string} Object${count === 1 ? '' : 's'}`;
 };
 
-const BucketsRowStatus: React.FC<BucketsRowStatusProps> = React.memo(({ status, link, error }) => (
-  <div className="nb-buckets-card__row-status-item">
-    {error || _.isNil(status) ? (
-      <span className="co-dashboard-text--small nb-buckets-card__row-subtitle">Not available</span>
-    ) : Number(status) > 0 ? (
-      <>
-        <a href={link} style={{ textDecoration: 'none' }} target="_blank" rel="noopener noreferrer">
-          <RedExclamationCircleIcon className="co-dashboard-icon nb-bucket-card__status-icon" />
-          <span className="nb-buckets-card__row-status-item-text">{status}</span>
-        </a>
-      </>
-    ) : null}
-  </div>
-));
+// Displays count of erroneous buckets due to issues not externalized as phase
+const OtherFailure: React.FC<BucketFailureItemProps> = React.memo(({ link, status }) => {
+  return (
+    <div className="nb-buckets-card__buckets-failure-status-item">
+      <a href={link} style={{ textDecoration: 'none' }} target="_blank" rel="noopener noreferrer">
+        <RedExclamationCircleIcon />
+        <span className="nb-buckets-card__buckets-failure-status-count">{status}</span>
+      </a>
+    </div>
+  );
+});
 
-const BucketsRow: React.FC<BucketsRowProps> = React.memo(
-  ({ bucketsCount, title, objectsCount }) => {
-    const subtitle: string = _.isNil(objectsCount)
-      ? 'Not available'
-      : formatCount(Number(objectsCount));
-    return (
-      <div className="nb-buckets-card__row-title">
-        <div>{_.isNil(bucketsCount) ? title : pluralize(Number(bucketsCount), title)}</div>
-        <div className="co-dashboard-text--small nb-buckets-card__row-subtitle">{subtitle}</div>
-      </div>
-    );
+// Displays count of erroneous buckets due to failure in provision
+const ProvisioningFailure: React.FC<BucketFailureItemProps> = React.memo(({ link, status }) => {
+  return (
+    <div className="nb-buckets-card__buckets-failure-status-item">
+      <Link to={link} style={{ textDecoration: 'none' }}>
```

Comment:
Any reason for not using `class` here? I can see that `Link` accept `className` prop.

Reply:
Actually it was getting overriden hence used inline styling.

Reply:
if that's the case, you can make the selector more specific. I don't think, it should be an issue here.

Reply:
You mean something like this:
```
.nb-buckets-card__buckets-failure-status-item {
  margin-left: var(--pf-global--spacer--sm);
  a {
    text-decoration: none;
  }
}
```
Well iniline css has highest specificity, hence used.

Reply:
I know that inline styles have highest specificity but adding inline style is not considered as a good practice as
- It can become hard to maintain as the complexity grows up
- Can't be reused
- It does not separate out the style information from content.

The above approach looks good and should be used.

Reply:
okay sending..